### DISCUSSION TITLE
Linking non-markdown files (PDFs, videos, etc) and GX support?

### DISCUSSION THREAD

**Discussion by justlvb** [epwalsh/obsidian.nvim#603] [Q&A]
I believe autocomplete for files or at least opening file links using "gx" should be supported.

`[video link](/home/user/video123.mp4)`
Is it possible to have auto complete for links to non md files? As an option, or only popping up when typing in the parantheses portion?

Also gx support for file paths would be great.

**Comment by epwalsh** [comment]
Hey @justlvb, for completing to non-note files I would recommend using `cmp-path`.

**Reply by justlvb** [reply]
Any advice on how I would do that? You referred to this one: https://github.com/hrsh7th/cmp-path

**Comment by amyrd** [comment]
@justlvb sorry not a solution but a question, are you able to open attachments (pdfs, mp4, etc) through this plugin?

**Reply by justlvb** [reply]
No not directly, though you can highlight the path using vi( and then gx.
